Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -2591,6 +2610,62 @@ def suppress_alert_command(client: Client, args: Dict[str, Any]) -> CommandResul
     )
 
 
+def get_alert_group_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+
+    returns the alert group id
+
+    """
+    agent_id = str(args.get('agentId', ''))
+    alert_id = args.get('alertId')
+
+    try:
+        alert_group_ids = client.get_alert_group(agent_id)
+    except Exception as e:
+        if '404' in str(e):
+            raise ValueError(f"Alerts of host {agent_id} Not Found")
+
+    alert_groups = {}
+    for entry in alert_group_ids['data']['entries']:
+        alert_groups.update({str(entry['last_alert']['_id']): entry['_id']})
+    return alert_groups[alert_id]
+
+
+def acknowledge_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+
+    returns a success message to the war room
+
+    """
+    agent_id = str(args.get('agentId', ''))
+    alert_id = args.get('alertId')
+    comment = str(args.get('comment', ''))
+
+    try:
+        alert_group_ids = client.get_alert_group(agent_id)
+
+        alert_groups = {}
+        for entry in alert_group_ids['data']['entries']:
```

Comment:
What will happen if the alert_group_ids['data']['entries'] will not return in the response you should get a keyError. please handle it.